src/components/data/ExportDataModal.js

```diff
@@ -138,12 +138,16 @@ const ExportDataModal = ({ onDismiss, selectedDataType, selectedEntities, runnin
         content: 'The following data is linked to entries which already exist in the selected workspace. You may re-link the following data to the existing entries by clicking COPY.'
       }),
       h(FormLabel, ['Entries selected']),
-      ...Utils.cond(
-        [!!additionalDeletions.length, () => displayEntities(additionalDeletions, runningSubmissionsCount, true)],
-        [!!hardConflicts.length, () => displayEntities(hardConflicts, runningSubmissionsCount, true)],
-        [!!softConflicts.length, () => displayEntities(softConflicts, runningSubmissionsCount, true)],
-        () => displayEntities(selectedEntities, runningSubmissionsCount, false)
-      ),
+      // Size the scroll container to cut off the last row to hint that there's more content to be scrolled into view
```

Comment:
Wow. I dub thee a master of usability. Go forth unto the world.

Reply:
I can't take credit for this idea: https://github.com/DataBiosphere/terra-ui/pull/2828#discussion_r810261985